source/documentation/how-to/hosted-engine.html.md

```diff
@@ -57,6 +57,33 @@ On the VM:
 
 When the engine-setup has completed on the VM, return to the host and complete the configuration. Your hosted engine VM is up and running!
 
+## **Fresh Install (via Web UI)**
+
+Self-Hosted Engine can be deployed also via a web UI inroduced by cockpit.
+To deploy the self-hosted engine using the Cockpit user interface, follow these steps:
+
+1. Install the cockpit-ovirt-dashboard package:
+
+        # yum install cockpit-ovirt-dashboard
+       
+2. Start and enable cockpit:
+
+        # systemctl enable cockpit
+        # systemctl start cockpit
+        
+3. Open 9090 port:
+
+        # firewall-cmd --add-port=9090/tcp --permanent
+        # firewall-cmd --reload
+        
+4. Log in to the UI at https://HostIPorFQDN:9090
```

Comment:
Please add something like:

 By default, you should get a warning/popup from your browser about a self-signed certificate, unknown issuer, or something like that. Accept it, or see <link> for more details

Where <link> is some link you can hopefully find about cockpit and ssl/https/etc.